docs/experience_date_search/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+title: Experience date search
+has_children: true
+nav_order: 8
+has_toc: false
+---
+
+# Experience date search
+
+Experience date search enables you to filter a Creator's experiences with a separate result for each departure date of each experience.
+
+The experience date search is executed using the [experience_date_search tag]({% link docs/reference/tags/experience_date_search_tag/index.md %}). The tag returns an `items` array of [ExperienceDates]({% link docs/reference/objects/product/experience_date.md %}), a `paginate` [Pagination]({% link docs/reference/objects/pagination.md %}) object and a `search` [Search]({% link docs/reference/objects/search_query.md %}) object.
```

Comment:
stating the obvious but experience_date link currently broken

Reply:
Yeah exactly, it's dependent on this one:
https://github.com/easolhq/easolhq.github.io/pull/124

I might merge just hold on merge until that one is done.

Reply:
other done, merge this whenever your heart desires